tracer/test/Datadog.Trace.Tests/Propagators/DistributedPropagatorTests.cs

```diff
@@ -54,7 +54,8 @@ static DistributedPropagatorTests()
     {
         Propagator = new SpanContextPropagator(
             injectors: null,
-            extractors: new IContextExtractor[] { DistributedContextExtractor.Instance });
+            extractors: new IContextExtractor[] { DistributedContextExtractor.Instance },
+            Tracer.Instance.Settings.PropagationExtractFirst);
```

Comment:
Note from our offline chat: Let's default to `false` here.

```suggestion
            false);
```